### ISSUE TITLE
Rmatrix error

### ISSUE THREAD

**Issue by meryembel** [flexible-atomic-code/fac#694]
Hi, when i run rmx.py I have error :
`python2 rmx.py 
FAC was not compiled with MPI or OpenMP support.
Multi-threading disabled.
rmatrix boundary: 0 1.25e-07 979 6.60559
too large kappa, enlarge korbmap: 50 >= 50
`

**Comment by mfgu** [comment]
use SetOrbMap(nmax) in the beginning of the input, where nmax=2*max_orb_angular_momentum you have.

**Comment by meryembel** [comment]
I have the same error
`FAC was not compiled with MPI or OpenMP support.
Multi-threading disabled.
rmatrix boundary: 0 1.25e-07 979 6.60559
too large kappa, enlarge korbmap: 60 >= 60`

**Comment by mfgu** [comment]
try SetOrbMap(61).  nmax=2*kmax+1

**Comment by meryembel** [comment]
yes ,it runs, but when i run cs.py i also get an error :`python2 cs.py 
FAC was not compiled with MPI or OpenMP support.
Multi-threading disabled.
malloc error: 14800146824 7252289296
Erreur de segmentation (core dumped)
`

**Comment by mfgu** [comment]
i cannot reproduce this error. since you build does not include OpenMP, can you change cs.py to remove the InitializeMPI and FinalizeMPI, see if it still seg faults.

**Comment by meryembel** [comment]
After some effort, the code is finally running.  However,  the energy levels and collision strengths calculated using the distorted wave method differ from those obtained with the R-matrix method.
` fac.SetAtom('O')
# 1s shell is closed
fac.Closed('1s')
fac.Config('2*1', group = 'n2')
fac.Config('3*1', group = 'n3')
#fac.Config('4*1', group = 'n4')
#fac.Config('5*1', group = 'n5')


# Self-consistent iteration for optimized central potential
fac.ConfigEnergy(0)
fac.OptimizeRadial('n2')
fac.ConfigEnergy(1)
fac.Structure('ne.lev.b')
fac.MemENTable('ne.lev.b')
fac.PrintTable('ne.lev.b', 'ne.lev', 1)
e = [1.8798989520E+02]
fac.SetCEGrid(e)
fac.SetUsrCEGrid(e)
fac.CETable('ne.ce.b', [0], [1])
fac.PrintTable('ne.ce.b', 'ne.ce', 1)
` 
`     0  1      1  1  1.2035E+01 1
 5.9998E-01  1.4307E+00  1.2730E+04
 1.8799E+02  2.8817E+00  8.6204E+02`
and for Rmatrix : `from pfac.fac import *
import time


"""
Calculate R-matrix basis and surface amplitudes for He-like O.
Targets include 1s2, 1s1 2*1, and 1s1 3*1 configurations.
"""

#ConvertToSFAC('t.sf')
SetAtom('O')

# Target configurations
Config('g0', '1s2')
Config('g1', '1s1 2*1')
Config('g2', '1s1 3*1')

# Correlation configurations
Config('g3', '1s2 2*1')
#Config('g3', '1s2 3*1')
#Config('g3', '1s1 2*2')
#Config('g3', '1s1 2*1 3*1')
#Config('g3', '1s1 3*2')

# disable Breit interaction, and nuclear recoil effects.
SetBreit(0)
SetMS(0,0)
SetSE(0)
SetVP(0)

# use maximum number of grid points, important.
SetRadialGrid(3000,1.4,300,0)

OptimizeRadial('g0')

Structure('tb.en', ['g0','g1','g2'])
Structure('tb.en', ['g3'])

MemENTable('tb.en')
PrintTable('tb.en', 'ta.en')
BasisTable('bas.d')

# specify targets correlation states.
RMatrixTargets(['g0','g1','g2'], ['g3'])

nb = 5
kmax = 2
# slater integrals with l >= 12 are treated with no exchange.
SetSlaterCut(12,12)
SetOrbMap(0)
# boundary radius of the first R-matrix zone.
# it enclose the 1s,2s,2p,3s,3p,3d wavefunctions
# with amplitude > 1E-6
RMatrixBoundary(0, 1e-6, 0.0)
GetPotential('pot.d')

# calculate basis functions, with maximum angular momemtum kmax,
# and nb functions per-kappa.
RMatrixBasis('rmx.b0', kmax, nb)

# R-matrix surface amplitudes.
RMatrixSurface('rmx.d0')

# boundary radius of the second zone.
# it starts from the end of first zone, to the radius 13.0 a.u.
RMatrixBoundary(-1, 13.0, 0.0)

# calculate basis functions in the second zone.
RMatrixBasis('rmx.b1', kmax, nb)
# R-matrix surface amplitudes.
RMatrixSurface('rmx.d1')

"""
Calculate the collision strengths using the R-matrix method.
"""

#ConvertToSFAC('trmx.sf')


t0 = time.time()

# list of R-matrix basis files, calculated with rmx.py
b = ['rmx.b0', 'rmx.b1']

# list of R-matrix surface files.
r = ['rmx.d0', 'rmx.d1']

# call RMatrixCE, calculate collsion strength at E=750--1.5E3 eV,
# in 100 eV step. Results are in r1.d
RMatrixCE('r1_m2_100_110_1.d', b, r, 27.994, 28.994, 0.1, 0)

t1 = time.time()
print('Time = %12.5E'%(t1-t0))
  

` `#   0   0   1   2  5.61412E+02     31   3
  0   1 2.7994000000E+01  2.79940000E+01 -5.3341819845E+02  0.00000E+00
  0   1 2.8027333333E+01  2.80273333E+01 -5.3338486511E+02  0.00000E+00`
